chicagobub/index.html

```diff
@@ -1,30 +1,29 @@
-<!doctype html>
+<!DOCTYPE html>
 <html>
-<head>
-  <meta charset="utf-8">
+  <head>
+    <meta charset="utf-8" />
 
-  <meta name="viewport" content="width=device-width,initial-scale=1">
-  <link rel="stylesheet" href="css/reset.css">
-  <link rel="stylesheet" href="css/bubble_chart.css">
-</head>
-<body>
-  <div class="container">
-    <h1>Shrinking Middle Class in Chicago</h1>
-    <div id="toolbar">
-      <a href="#" id="all" class="button active">All</a>
-      <a href="#" id="year" class="button">1970</a>
-      <a href="#" id="year" class="button">1980</a>
-      <a href="#" id="year" class="button">1990</a>
-      <a href="#" id="year" class="button">2000</a>
-      <a href="#" id="year" class="button">2010</a>
-      <a href="#" id="year" class="button">2016</a>
+    <meta name="viewport" content="width=device-width,initial-scale=1" />
+    <link rel="stylesheet" href="css/reset.css" />
+    <link rel="stylesheet" href="css/bubble_chart.css" />
+  </head>
+  <body>
+    <div class="container">
+      <h1>Shrinking Middle Class in Chicago</h1>
+      <div id="toolbar">
+        <a href="#" id="all" class="button active">All</a>
+        <a href="#" id="1970" class="button">1970</a>
+        <a href="#" id="1980" class="button">1980</a>
+        <a href="#" id="1990" class="button">1990</a>
+        <a href="#" id="2000" class="button">2000</a>
+        <a href="#" id="2010" class="button">2010</a>
+        <a href="#" id="2016" class="button">2016</a>
```

Comment:
important change here is to give all the year buttons unique `id` values, instead of just 'year'